packages/salesforcedx-vscode-apex/src/commands/metadataOrchestrator.ts

```diff
@@ -177,18 +173,22 @@ export class MetadataOrchestrator {
       `\n\`\`\`\n${endOfPromptTag}\n${assistantTag}`;
     console.log('input = ' + input);
     let result;
-    let documentContents;
+    let documentContents = '';
+    let tries = 0;
     try {
       const apiClient = await this.getAiApiClient();
-      result = await apiClient.naturalLanguageQuery({
-        prefix: '',
-        suffix: '',
-        input,
-        commandSource: CommandSource.NLtoCodeGen,
-        promptId: 'generateOpenAPIv3Specifications'
-      });
-      documentContents = result[0].completion;
-      if (documentContents.includes('try again')) throw new Error(documentContents);
+      while (!documentContents.startsWith('yaml') && tries < 10) {
```

Comment:
Why the retries?

Reply:
It's just a temporary workaround to not have to run the command multiple times until the doc is finally generated